lib/config.ts

```diff
@@ -319,7 +269,7 @@ export interface Config {
    * `beforeLaunch` is run, and before any driver is set up. If this is
    * specified, both capabilities and multiCapabilities will be ignored.
    */
-  getMultiCapabilities?: any;
+  getMultiCapabilities?: () => q.IWhenable<Array<Capabilities>>;
```

Comment:
Could the type here be `Promise<Array<Capabilities>>`? `q.IWhenable` is gross to encounter while reading documentation.